physics/GFS_rrtmgp_cloud_mp.F90

```diff
@@ -484,47 +485,48 @@ subroutine cloud_mp_SAMF(nCol, nLev, t_lay, p_lev, p_lay, qs_lay, relhum,
          nCol,          & ! Number of horizontal grid points
          nLev             ! Number of vertical layers
     real(kind_phys), intent(in) :: &
-         con_g,         & ! Physical constant: gravitational constant 
-         con_ttp,       & ! Triple point temperature of water (K)
+         con_g,         & ! Physical constant: gravity         (m s-2)
+         con_ttp,       & ! Triple point temperature of water  (K)
          alpha0           !
     real(kind_phys), dimension(:,:),intent(in) :: &
-         t_lay,         & ! Temperature at layer centers (K)
-         p_lev,         & ! Pressure at layer interfaces (Pa)
-         p_lay,         & !
-         qs_lay,        & !
-         relhum,        & !
-         cnv_mixratio     ! Convective cloud mixing-ratio (kg/kg)
+         t_lay,         & ! Temperature at layer-centers       (K)
+         p_lev,         & ! Pressure at layer-interfaces       (Pa)
+         p_lay,         & ! Presure at layer-centers           (Pa)
+         qs_lay,        & ! Specific-humidity at layer-centers (kg/kg)
+         relhum,        & ! Relative-humidity                  (1)
+         cnv_mixratio     ! Convective cloud mixing-ratio      (kg/kg)
     ! Outputs
     real(kind_phys), dimension(:,:),intent(inout) :: &
          cld_cnv_lwp,   & ! Convective cloud liquid water path
          cld_cnv_reliq, & ! Convective cloud liquid effective radius
          cld_cnv_iwp,   & ! Convective cloud ice water path
          cld_cnv_reice, & ! Convective cloud ice effecive radius
-         cld_cnv_frac     ! Convective cloud-fraction (1)
+         cld_cnv_frac     ! Convective cloud-fraction
     ! Local
     integer :: iCol, iLay
-    real(kind_phys) :: tem1, deltaP, clwc
+    real(kind_phys) :: tem0, tem1, deltaP, clwc
 
+    tem0 = 1.0e5/con_g
     do iLay = 1, nLev
        do iCol = 1, nCol
           if (cnv_mixratio(iCol,iLay) > 0._kind_phys) then
              tem1   = min(1.0, max(0.0, (con_ttp-t_lay(iCol,iLay))*0.05))
              deltaP = abs(p_lev(iCol,iLay+1)-p_lev(iCol,iLay))*0.01
-             clwc   = max(0.0, cnv_mixratio(iCol,iLay)) * con_g * deltaP
-             cld_cnv_iwp(iCol,iLay) = clwc * tem1
-             cld_cnv_lwp(iCol,iLay) = clwc - cld_cnv_iwp(iCol,iLay)
+             clwc   = max(0.0, cnv_mixratio(iCol,iLay)) * tem0 * deltaP
```

Comment:
You modified from con_g to 1.0e5/con_g. What is 1.0e5?

Reply:
@ChunxiZhang-NOAA This was a bug fix identified by @Qingfu-Liu back in June.

Reply:
@dustinswales Thanks for clarifying.